tests/test_project.py

```diff
@@ -334,7 +355,16 @@ def test_find_jobs_arithmetic_operators(self):
 
     def test_find_jobs_logical_operators(self):
         for i in range(10):
-            self.project.open_job({'a': i, 'b': {'c': i}}).init()
+            job = self.project.open_job({'a': i, 'b': {'c': i}})
+            job.init()
```

Comment:
```suggestion
            job = self.project.open_job({'a': i, 'b': {'c': i}}).init()
```

Reply:
I tried it. But,`self.project.open_job(get_sp(i)).init()` returns `None`.